apps/envited-x-data-space/modules/AddAssets/UploadAssetItem.utils.ts

```diff
@@ -0,0 +1,26 @@
+import { isEmpty } from 'ramda'
```

Comment:
Let's put this in a common place. This does not seem specific to UploadAssetItem.